hera_mc/args_test.py

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2016 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""This is meant to hold utility scripts for handling parts and connections
+
+"""
```

Comment:
I didn't change the comment - it is just meant to be an import module to allow testing of cm_handling and cm_hookup within python